### PR TITLE
Update Epic ticker to show number of supporters

### PR DESCRIPTION
**Pull request by tomrf1**
## What does this change?
The existing Epic ticker displays money raised during a campaign.
In an upcoming moment we will display the number of supporters instead.
The ticker will now support both modes.
A new url is used to fetch data for the supporter count.

I have added a `TickerSettings` type to expand how we configure the epic. It allows us to configure the copy.
In the long term we will configure this from the epic tool.

Note - the ticker in `contributions-service` (for DCR) will need to support this configuration as well.
See also - the change on the contributions landing page: https://github.com/guardian/support-frontend/pull/2522

### Before goal
Screen Shot 2020-06-04 at 12 10 33

### After goal
Screen Shot 2020-06-04 at 12 10 03

### Before goal
Screen Shot 2020-06-04 at 12 30 22

### After goal
Screen Shot 2020-06-04 at 12 31 18

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/ticker.js [hunk 1]
```diff
@@ -108,69 +145,91 @@ const populateGoal = (parentElement: HTMLElement, tickerType: TickerType) => {
...
-export const initTicker = (
+const defaultSettings: TickerSettings = {
+    endType: 'unlimited',
+    countType: 'people',
+    copy: {
+        countLabel: 'supporters in Australia',
+        goalReachedPrimary: 'We\'ve hit our goal!',
+        goalReachedSecondary: 'but you can still support us',
+    },
+    currencySymbol: '$',
```
Comment (tjmw) [inline static/src/javascripts/projects/common/modules/commercial/ticker.js:219]: Does it make sense to have a default `currencySymbol` here if the `countType` is people? I think it won't be used either way, but _could_ be confusing?

File: static/src/javascripts/__flow__/types/ab-tests.js [hunk 2]
```diff
@@ -31,7 +32,8 @@ declare type EpicVariant = Variant & {
     componentName: string,
     template: EpicTemplate,
     classNames: string[],
-    showTicker: boolean,
+    showTicker?: boolean,    // Deprecated, use tickerSettings instead
```
Comment (tjmw) [inline static/src/javascripts/__flow__/types/ab-tests.js:35]: Once the relevant changes have been made to the epic admin tool, support for `showTicker` could be removed right? (Just want to make sure I understand correctly!)

**Review by tjmw — APPROVED** [review]
💯This looks good! I left a couple of minor suggestions/questions to take or leave.

Thanks for the screenshots in the description! In the final one (money, after goal) is it right that the text under the amount reads `contributors` instead of `contributed`?

File: static/src/javascripts/__flow__/types/ab-tests.js (hunk 2 above)
Comment (tomrf1) [reply static/src/javascripts/__flow__/types/ab-tests.js:35]: Yes that's right, I've left it here as an optional field for backwards compatibility

File: static/src/javascripts/projects/common/modules/commercial/ticker.js (hunk 1 above)
Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/commercial/ticker.js:219]: You're right!
I started out with this as optional, but decided it's simpler to make it mandatory than do extra validation

**Comment by tomrf1** [comment]
> In the final one (money, after goal) is it right that the text under the amount reads `contributors` instead of `contributed`?

good spot, it should be `contributed`. I actually hardcoded this for testing so there's nothing to fix

**Pull request merged**